### PR TITLE
admission config: fix decoding error for legacy config without Kind

### PR DESCRIPTION
**Pull request by sttts**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1452092

### CODE REVIEW

**Comment by sttts** [comment]
[test]

File: pkg/cmd/util/pluginconfig/config.go [hunk 1]
```diff
@@ -69,27 +70,34 @@ func GetAdmissionConfigurationConfig(pluginName string, cfg configapi.AdmissionP
...
 		f, err := os.Open(cfg.Location)
 		if err != nil {
 			return "", err
 		}
-		cfgObj, err = latest.ReadYAML(f)
+		data, err := ioutil.ReadAll(f)
+		if err != nil {
+			return "", err
+		}
+		cfgJSON, err = kyaml.ToJSON(data)
```
Comment (liggitt) [inline pkg/cmd/util/pluginconfig/config.go]: so this just inlines json/yaml from an external file into the config?

**Comment by liggitt** [comment]
this change LGTM, though I would still expect an external config file without a kind/apiVersion (as referenced in the original bz) to fail decoding

**Comment by sttts** [comment]
@liggitt it doesn't fail inline in the master config. Why should a file be different?

**Comment by liggitt** [comment]
I'd expect it to... isn't it supposed to decode to a runtime.Object internally? how does it do that without a kind/apiVersion?

**Comment by sttts** [comment]
@liggitt of course, it's decoded to runtime.Object, but with an explicit Go type, compare https://github.com/openshift/origin/blob/c1e03fe1b66f27b8e920ae996d525e15ed449161/pkg/build/admission/overrides/admission.go#L20-L20. I haven't traced the code, but the json serializer (vendor/k8s.io/apimachinery/pkg/runtime/serializer/json/json.go:120) looks very forgiving about missing Kind in this case.

File: pkg/cmd/util/pluginconfig/config.go (hunk 1 above)
Comment (sttts) [inline pkg/cmd/util/pluginconfig/config.go]: yes, read the file and inlines the json content typeless via RawExtension.

**Comment by liggitt** [comment]
lgtm [merge][severity:bug]

**Pull request merged**